lib/decanter/parser/array_parser.rb

```diff
@@ -10,7 +10,7 @@ class ArrayParser < ValueParser
         # Fetch parser classes for provided keys
         parse_each = options.fetch(:parse_each, :pass)
         item_parsers = Parser.parsers_for(Array.wrap(parse_each))
-        unless item_parsers.all? { |parser| parser <= ValueParser }
+        unless item_parsers.all? { |parser| parser <= ValueParser || parser <= PassParser} 
```

Comment:
I had to make `PassParser` inherit from `Base` so that it wouldn't throw an error when an array was passed to it.